scripts/docker/docker-compose.yml

```diff
@@ -22,6 +22,11 @@ services:
       - PCS_AUTH_ISSUER
       - PCS_AUTH_AUDIENCE
       - PCS_TWIN_READ_WRITE_ENABLED
+      - PCS_LOG_LEVEL
+      - PCS_SUBSCRIPTION_DOMAIN
```

Comment:
Are these used anywhere in the service ?
- PCS_SUBSCRIPTION_DOMAIN
- PCS_SUBSCRIPTION_ID
- PCS_RESOURCE_GROUP

Reply:
yes, see `appsettings.ini`